Lib/pathlib.py

```diff
@@ -479,6 +260,113 @@ def __reduce__(self):
         # when pickling related paths.
         return (self.__class__, tuple(self._parts))
 
+    @classmethod
+    def _split_extended_path(cls, s):
+        prefix = ''
+        if s.startswith(_WIN_EXT_NAMESPACE_PREFIX):
+            prefix = s[:4]
+            s = s[4:]
+            if s.startswith('UNC\\'):
+                prefix += s[:3]
+                s = '\\' + s[3:]
```

Comment:
```suggestion
                extra_prefix, sep, rest = s.partition(r'\')
                prefix += extra_prefix
                s = sep + rest
```

Reply:
I'd like to leave this code alone (aside from moving it) as it'll be removed in #91882 - see my top-level comment. Hope that makes sense! Resolving this thread...